partcad-cad-freecad/partcad_freecad/client.py

```
@@ -0,0 +1,149 @@
+        [executable, "--stdio", *extra_args],
+        stdin=subprocess.PIPE,
+        stdout=subprocess.PIPE,
+        cwd=cwd,
+        creationflags=_NO_WINDOW,
+    )
+
+    def closer():
+        try:
+            proc.stdin.close()
+        except Exception:  # pylint: disable=broad-except
+            pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)